tests/delta-merge-test/query/expr/round_with_frac.test

```diff
@@ -1,466 +0,0 @@
-=> drop table if exists default.i64
-=> DBGInvoke __drop_tidb_table(default, u64)
-=> drop table if exists default.u64
-=> DBGInvoke __drop_tidb_table(default, t)
-=> drop table if exists default.t
-
-# Data.
-=> DBGInvoke __mock_tidb_table(default, i8, 'a Nullable(Int8)')
-=> DBGInvoke __mock_tidb_table(default, u8, 'a Nullable(UInt8)')
-=> DBGInvoke __mock_tidb_table(default, i32, 'a Nullable(Int32)')
-=> DBGInvoke __mock_tidb_table(default, u32, 'a Nullable(UInt32)')
-=> DBGInvoke __mock_tidb_table(default, i64, 'a Nullable(Int64)')
-=> DBGInvoke __mock_tidb_table(default, u64, 'a Nullable(UInt64)')
-=> DBGInvoke __mock_tidb_table(default, t, 'i Nullable(Int64)')
-
-=> DBGInvoke __refresh_schemas()
-
-=> DBGInvoke __put_region(1, 0, 100, default, i8)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 1, 0)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 2, 1)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 3, -1)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 4, 49)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 5, 50)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 6, -49)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 7, -50)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 8, 127)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 9, -128)
-=> DBGInvoke __raft_insert_row(default, i8, 1, 10, NULL)
-
-=> DBGInvoke __put_region(2, 0, 100, default, u8)
-=> DBGInvoke __raft_insert_row(default, u8, 2, 1, 0)
-=> DBGInvoke __raft_insert_row(default, u8, 2, 2, 4)
-=> DBGInvoke __raft_insert_row(default, u8, 2, 3, 5)
-=> DBGInvoke __raft_insert_row(default, u8, 2, 4, 49)
-=> DBGInvoke __raft_insert_row(default, u8, 2, 5, 50)
-=> DBGInvoke __raft_insert_row(default, u8, 2, 6, 255)
-=> DBGInvoke __raft_insert_row(default, u8, 2, 7, NULL)
-
-=> DBGInvoke __put_region(3, 0, 100, default, i32)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 1, 0)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 2, 4)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 3, 5)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 4, -4)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 5, -5)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 6, 499999999)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 7, 500000000)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 8, -499999999)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 9, -500000000)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 10, 2147483647)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 11, -2147483648)
-=> DBGInvoke __raft_insert_row(default, i32, 3, 12, NULL)
-
-=> DBGInvoke __put_region(4, 0, 100, default, u32)
-=> DBGInvoke __raft_insert_row(default, u32, 4, 1, 0)
-=> DBGInvoke __raft_insert_row(default, u32, 4, 2, 4)
-=> DBGInvoke __raft_insert_row(default, u32, 4, 3, 5)
-=> DBGInvoke __raft_insert_row(default, u32, 4, 4, 499999999)
-=> DBGInvoke __raft_insert_row(default, u32, 4, 5, 500000000)
-=> DBGInvoke __raft_insert_row(default, u32, 4, 6, 4294967295)
-=> DBGInvoke __raft_insert_row(default, u32, 4, 7, null)
```

Comment:
I have a little doubt, after the migration, the relevant logic of the storage layer will not be able to go.
Is this expected?

Reply:
> I have a little doubt, after the migration, the relevant logic of the storage layer will not be able to go. Is this expected?

Yes, IMO all the migrated tests don't need to use storage layer.

Reply:
These tests under `/query/{executor,expr}` should focus on the compute layer and not care about the storage layer.